source/IAccessibleHandler.py

```diff
@@ -569,6 +569,9 @@ def winEventCallback(handle,eventID,window,objectID,childID,threadID,timestamp):
 				window=tempWindow
 
 		windowClassName=winUser.getClassName(window)
+		# Modern IME candidate list windows fire menu events which confuse us and can't be used properly inconjunction with input composition support

```

Comment:
This line is pretty long. Could you please split it?

```suggestion
		# Modern IME candidate list windows fire menu events which confuse us
		# and can't be used properly in conjunction with input composition support.
```